packages/fxa-auth-server/lib/routes/subscriptions/paypal.ts

```diff
@@ -158,6 +160,11 @@ export const paypalRoutes = (
             token: isA.string().required(),
           }),
         },
+        validate: {
+          payload: {
+            currencyCode: isA.string().required(),
```

Comment:
I was originally wondering why the client is choosing the currency, but I see the only alternative at this point is to have the plan id sent. Since we're going to force a currency match later anyways, it doesn't seem to matter much. 😄